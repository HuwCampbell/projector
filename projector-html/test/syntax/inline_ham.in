<ul>Ham (<code>jamon</code>)</ul>