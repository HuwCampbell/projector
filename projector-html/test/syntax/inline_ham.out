Template
  ()
  Nothing
  (TENode
     ()
     (THtml
        ()
        [ TElement
            ()
            (TTag () "ul")
            []
            (THtml
               ()
               [ TPlain () TPlainText { unTPlainText = "Ham" }
               , TWhiteSpace () 1
               , TPlain () TPlainText { unTPlainText = "(" }
               , TElement
                   ()
                   (TTag () "code")
                   []
                   (THtml () [ TPlain () TPlainText { unTPlainText = "jamon" } ])
               , TPlain () TPlainText { unTPlainText = ")" }
               ])
        ]))